Formula/b/build2.rb

```diff
@@ -1,35 +1,19 @@
 class Build2 < Formula
   desc "C/C++ Build Toolchain"
   homepage "https://build2.org"
-  url "https://download.build2.org/0.16.0/build2-toolchain-0.16.0.tar.xz"
-  sha256 "23793f682a17b1d95c80bbd849244735ed59a3e27361529aa4865d2776ff8adc"
+  url "https://download.build2.org/0.17.0/build2-toolchain-0.17.0.tar.xz"
+  sha256 "3722a89ea86df742539d0f91bb4429fd46bbf668553a350780a63411b648bf5d"
   license "MIT"
 
   livecheck do
     url "https://download.build2.org/toolchain.sha256"
     regex(/^# (\d+\.\d+\.\d+)(?:\+\d+)?$/i)
   end
 
-  bottle do
-    sha256 arm64_sonoma:   "cfcd09aa221a08dd5b190e1d471822946c7e08210f74dc3be4ef8f89f088d937"
-    sha256 arm64_ventura:  "fc0c8d37ea3a336f182133daeb135699a45b34d8fa02af56dbc104fff33d6f3c"
-    sha256 arm64_monterey: "93e00c45fcac69a862a26b5e39004c1217675f6a482812107b27d85ff9ab2318"
-    sha256 sonoma:         "aa3e700461c6a0c2052cb3d89a78f9f37bd65e9b8a6fcb9191a5a0b2f3d29540"
-    sha256 ventura:        "61a7a28750af25b609ed325f532773d2624e23ca6ebfb616574cce759f8a4bc0"
-    sha256 monterey:       "1f90f3b72cadd6d429164f3db7f49a200e72da2faa1316a0f113bbe2d264b287"
-    sha256 x86_64_linux:   "92b0ca8b79c117d3e4b0462cf0ab7e1d2d4fffe7c17331e02a7e0b336ac505ab"
-  end
-
-  depends_on "pkgconf"
-
   uses_from_macos "curl"
   uses_from_macos "openssl"
-  uses_from_macos "sqlite"
```

Comment:
any ref for these dep updates?

Reply:
The background/details are here: https://lists.build2.org/archives/users/2024-June/001117.html

Related:

https://github.com/Homebrew/homebrew-core/pull/167330
https://github.com/build2/build2/issues/381